todo_app/app.py

```diff
@@ -1,21 +1,61 @@
-from flask import Flask, render_template, request,redirect
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
+from flask import Flask, render_template,redirect
+from todo_app.data.trello_items import createcard, getcards
 from todo_app.flask_config import Config
+import requests
+import json
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
+@app.route('/' , methods=['GET'])
+def alltodoitems():
+    response = getcards()
+    cards=response.json() #changing the python objects to json
+    Allitems=[]
+    for card in cards:
+        item=Item(card['id'],card['name'])
+        Allitems.append(item)
 
-@app.route('/')
-def index():
-     allitems=get_items()
-     return render_template("index.html",todo=allitems)
-     
+    return render_template("index.html",todo=Allitems)
 
 
-@app.route('/newitem' , methods=['POST'])
-def newitems():
-    Mynewtitle = request.form["title"] 
-    add_item(Mynewtitle)
+@app.route('/createnewcard' , methods=['POST'] )
+def createnewcard ():
+    response = createcard()
+    print(response.text)
     return redirect('/')
+
+    
+
+@app.route('/complete_item/<id>', methods=['POST'])
+def complete_item(id):
+    url =f"https://api.trello.com/1/cards/{id}"
+    params ={ 'key':os.getenv('TRELLO_KEY'),
+             'token':os.getenv('TRELLO_TOKEN'),
+            'idList':os.getenv('TRELLO_LISTID2')
+    }
+
+    headers = {
+        "Accept": "application/json"
+}
+
+    response = requests.request(
+        "PUT",
+        url,
+        headers=headers,
+        params=params
+    )
```

Comment:
Again minor, but thinking from a 'clean code' perspective, I would expect this logic to be in the `trello_items.py` file.